### DISCUSSION TITLE
essay/2024/06/03/depends-on-your-service

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#214] [Comments]
# essay/2024/06/03/depends-on-your-service

MVP는 어느 정도 수준으로 해야할까요? 초기 마케팅은 어떻게 해야하나요? 웹으로 만드는게 좋을까요, 앱으로 만드는 게 좋을까요? 리텐션이 좋은건지는 어떻게 알 수 있을까요? 저는 개발자인데 디자인은 어느 정도 수준으로 하는게 좋을가요?

https://jeho.page/essay/2024/06/03/depends-on-your-service.html

**Comment by hymi54** [comment]
안녕하세요, 사이드 프로젝트로 1인 개발을 준비하는 중입니다. 아직 아이디에이션 단계지만... 아무래도 눈에 보이는 게 있으면 더 동기부여가 될 것 같은데 디자인은 커녕 프런트도 손을 대본 적이 없어서 감이 잘 안오네요... ㅠㅠ 혹시 괜찮으시다면 와이어프레임 제작기? 같은 걸 부탁드려도 될까요? 아직 글을 찾지는 못했지만 혹시 이미 작성하셨다면,, ㅎㅎ 말씀해주시면 더 찾아보겠습니당
눈팅만 하다가 댓글을 달려고 하니 쫌 어색하기도 하네요 항상 좋은 글 감사해요~ ^^

**Reply by BenjaminKim** [reply]
와이어프레임을 만든 후 개발을 하지 않습니다. 그냥 머리 속에 UI 를 상상하고 종이에 대강 그려본 후 바로 코딩을 합니다.
이런 방식이 허접해보일 수도 있겠지만... 혼자 일하는 것과 함께 일하는 것은 다르니까요.

**Reply by hymi54** [reply]
오호... 그렇네요 결국 와이어프레임도 소통 수단의 일종이니까 바로 구현하는게 오히려 더 린하겠네요! 산출물 중심으로 일하다 보니 고정관념이 생겼나봐요 ㅎㅎ 좋은 말씀 감사해요~~
